src/main/java/com/google/sps/data/DBObject.java

```diff
@@ -0,0 +1,9 @@
+package com.google.sps.meltingpot.data;
+
+public class DBObject {
```

Comment:
I'm not sure on this one, @hari387 any comments?

Reply:
Yeah this is a nice singleton base class to have in case you want to return just an id. I'll put a comment on it in my branch now that this is merged already.